src/plugins/dashboard/public/application/utils/get_no_items_message.tsx

```diff
@@ -60,11 +62,9 @@ export const getNoItemsMessage = (
                 values={{
                   sampleDataInstallLink: (
                     <EuiLink
-                      onClick={() =>
-                        application.navigateToApp('home', {
```

Comment:
What about call `application.navigationToApp('import_sample_data')` directly? Then we won't need so much modifications.

Reply:
Oh I didn't know that. It's a much better way to do that, thanks!